src/components/ChartTypes.jsx

```diff
@@ -75,4 +78,5 @@ export default connect(({ chartType, state }) => ({
     setOpen            : chartType.menu.setOpen,
     showAggregateDialog: chartType.showAggregateDialog,
     Type               : chartType.type,
+    onChartTypeChanged : chartType.onChange,
```

Comment:
@MahboobehMohammadi it'd be better to choose another name to covey that it's adding the passed`onChange` function from parent to the `ChartTypeStore`.